nbproject/platform.properties

```diff
@@ -3,7 +3,6 @@ cluster.path=\
     ${nbplatform.active.dir}/harness:\
     ${nbplatform.active.dir}/platform
 disabled.modules=\
-    org.jdesktop.layout,\
```

Comment:
Was this by accident?

Reply:
This happened in constellation-adaptors when i changed it from an 'application' to a 'module collection' - most likely unintended here.

Reply:
Can confirm was unintended